scripts/build_coverage.sh

```diff
@@ -8,7 +8,6 @@ mvn compile
 cd ..
 cd ./web
 mvn compile
-cd ..
```

Comment:
Please add this line back. Revoking this fix will bring the same problem that we hit before PR #247 was merged.

Reply:
This mistake is fixed now.